src/test/java/com/exasol/releaserobot/validation/GitHubPlatformValidatorTest.java

```diff
@@ -98,7 +98,7 @@ void testValidateWorkflowFileExists() {
         when(branchContentMock.getSingleFileContentAsString(GITHUB_WORKFLOW_PATH)).thenReturn("I exist");
         final GitHubPlatformValidator validator = new GitHubPlatformValidator(branchContentMock, null,
                 this.validationReport);
-        validator.validateWorkflowFileExists();
+        validator.validateFileExists(GITHUB_WORKFLOW_PATH, "file");
```

Comment:
Personally I don't think testing the internal implementation is a good idea.

I would rather test the public API (ie, MavenPlatformValidator.validate()) and add different test cases to cover this one, and remove this test.

The reason is that, by testing your private implementation you cannot refactor the code, cause every time you change something, you have to also change your tests and hope the still work after.
This also couples your production code to your tests, and you might end up having fragile tests.

If you agree, we can leave this for a future refactor as well.

@redcatbear thoughts?